web/modules/custom/asset_distribution/src/Plugin/Field/FieldWidget/FileSizeCalculatorWidget.php

```diff
@@ -30,8 +30,14 @@ class FileSizeCalculatorWidget extends WidgetBase implements ContainerFactoryPlu
   /**
    * {@inheritdoc}
    */
-  public function __construct($plugin_id, $plugin_definition, FieldDefinitionInterface $field_definition, array $settings, array $third_party_settings,
-                              EntityFieldManagerInterface $entity_field_manager) {
+  public function __construct(
+      $plugin_id,
+      $plugin_definition,
+      FieldDefinitionInterface $field_definition,
+      array $settings,
+      array $third_party_settings,
+      EntityFieldManagerInterface $entity_field_manager
+  ) {
```

Comment:
This is weird, why split it out like this? I don't think I have seen this pattern in Drupal before. I would just leave it on one single line.